src/components/common/TmSessionExtension.vue

```diff
@@ -48,29 +46,37 @@
               </ul>
             </div>
             <div v-else>
-              Use the Lunie Chrome extension to securely store your keys, and
+              Use the Lunie Browser Extension to securely store your keys, and
               safely perform transactions.
             </div>
           </div>
-        </div>
-        <div class="session-footer">
-          <p class="ledger-install">
-            You can find the Lunie Chrome extension on the
-            <a href="">Google Play Store</a>.
-          </p>
+
+          <div class="session-footer">
+            <p class="ledger-install">
+              You can find the Lunie Browser Extension in the
+              <a
+                href="https://chrome.google.com/webstore/category/extensions"
```

Comment:
If we are going to link to the extension in the chrome store we may want to have the link as a config value to update all locations at once. It would avoid 404's down the road if we miss updating a link.

Reply:
nice idea. i would propose making a separate ticket for this task. would apply to many affiliate links - we could already do this for ledger too.